### PR TITLE
Send mail function

### PR DESCRIPTION
**Pull request by JackyRahman**
### Debio 263
url: [https://blocksphere2020.atlassian.net/browse/DBIO-263](https://blocksphere2020.atlassian.net/browse/DBIO-263)

### [Back End] Function send email for staking services
build module, service, template for function send mail notification 
how to use function ready in mail.service.ts beetween line 35 and 55

### CODE REVIEW

File: src/mail/templates/notification.hbs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<p>Public address: {{public_address}}</p>
+<p>Country: {{country}}</p>
+<p>State/Province:{{state}}</p>
+<p>City: {{city}}</p>
+<p>Amount: {{amount}} {{currency}}</p>
+{{!-- <p>
+    <a href="{{ url }}">Confirm</a>
+</p> --}}
+
+{{!-- <p>If you did not request this email you can safely ignore it.</p> --}}
```
Comment (kevinjanada) [inline src/mail/templates/notification.hbs:12]: mas ini hapus aja ya

File: src/app.module.ts [hunk 2]
```diff
@@ -45,9 +46,10 @@ require('dotenv').config(); // eslint-disable-line
     EscrowModule,
     SubstrateModule,
     CachingModule,
-    SubstrateIndexedDataModule,
+    // SubstrateIndexedDataModule,
```
Comment (kevinjanada) [inline src/app.module.ts:49]: Ini kenapa di comment ya mas?

**Review by kevinjanada — CHANGES_REQUESTED** [review]
Please review

**Comment by JackyRahman** [comment]
> Please review

Sudah saya update dan upstream mas hehe

**Review by kevinjanada — APPROVED** [review]
Looks good

**Review by kevinjanada — CHANGES_REQUESTED** [review]
Mas @JackyRahman tlg resolve conflict nya dulu yah.
Soalnya commit nya kepanjangan klo aku resolve di github langsung.

**Review by kevinjanada — APPROVED** [review]

**Pull request merged**